.circleci/config.yml

```diff
@@ -58,11 +72,14 @@ workflows:
       - "Run Unit Tests":
               requires:
                 - "Build Bittensor"
-      - "Push Production candidate to DockerHub":
+      - "Run Integration Tests":
               requires:
                 - "Build Bittensor"
-                - "Run Unit Tests"
-              filters:
-                branches:
-                  only:
-                    - master
+      #- "Push Production candidate to DockerHub":
```

Comment:
Should we be leaving comments in these commit? @parall4x @shibshib

Reply:
Leftover comment, fixed.